Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -89,9 +95,9 @@ void GLContextWidget::timer_event(Core::TimerEvent&)
 
     angle -= 0.01f;
     auto matrix = Gfx::translation_matrix(FloatVector3(0, 0, -8.5))
-        * Gfx::rotation_matrix(FloatVector3(1, 0, 0), angle)
-        * Gfx::rotation_matrix(FloatVector3(0, 1, 0), 0.0f)
-        * Gfx::rotation_matrix(FloatVector3(0, 0, 1), angle);
+        * Gfx::rotation_matrix(FloatVector3(1, 0, 0), rotate_x ? angle * rotation_speed : 0.f)
+        * Gfx::rotation_matrix(FloatVector3(0, 1, 0), rotate_y ? angle * rotation_speed : 0.f)
+        * Gfx::rotation_matrix(FloatVector3(0, 0, 1), rotate_z ? angle * rotation_speed : 0.f);
```

Comment:
It would be best if we used [`glRotate`](https://www.khronos.org/registry/OpenGL-Refpages/gl2.1/xhtml/glRotate.xml) instead of doing this manually.

Reply:
Another benefit of using `glRotate` is that we don't have to transpose the matrix, thanks for the suggestion :)